benchmark-scripts/benchmark.sh

```diff
@@ -22,13 +22,15 @@ show_help() {
            [--classification_disabled] 
            [ --ocr_disabled | --ocr [OCR_INTERVAL OCR_DEVICE] ] 
            [ --barcode_disabled | --barcode [BARCODE_INTERVAL] ]
-           [realsense_enabled]
+           [--realsense_enabled]
 
          Note: 
           1. dgpu.x should be replaced with targetted GPUs such as dgpu (for all GPUs), dgpu.0, dgpu.1, etc
           2. filesrc will utilize videos stored in the sample-media folder
           3. Set environment variable STREAM_DENSITY_MODE=1 for starting single container stream density testing
           4. Set environment variable RENDER_MODE=1 for displaying pipeline and overlay CV metadata
+          5. Stream density can take two numbers: first one is for target fps and
```

Comment:
suggest changing "two numbers" to "two parameters" and add that fps is a float value